### PR TITLE
[Refactor]: added endpoints folders structure to b-c composite adapters

### PR DESCRIPTION
**Pull request by bogos**
## Description

Added endpoints folders structure to b-c composite adapter

## Changes

Add endpoints folders in composite adapters

## Quality Assurance

- [ ] Ran `yarn changeset` if adapter source code was changed
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [ ] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [ ] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [ ] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [X ] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/composites/bitcoin-json-rpc/src/config/index.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { Requester, util } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/types'
 
-export const DEFAULT_ENDPOINT = 'difficulty'
+export const DEFAULT_ENDPOINT = 'getblockchaininfo'
```
Comment (boxhock) [inline packages/composites/bitcoin-json-rpc/src/config/index.ts:4]: Why was this changed?

File: packages/composites/bitcoin-json-rpc/src/endpoint/getblockchaininfo.ts [hunk 2]
```diff
@@ -1,19 +1,41 @@
 import * as JSONRPC from '@chainlink/json-rpc-adapter'
-import { Config, ExecuteWithConfig } from '@chainlink/types'
+import { Config, ExecuteWithConfig, InputParameters } from '@chainlink/types'
 import { Validator, Requester } from '@chainlink/ea-bootstrap'
 
 export const NAME = 'getblockchaininfo'
```
Comment (boxhock) [inline packages/composites/bitcoin-json-rpc/src/endpoint/getblockchaininfo.ts:5]: This one should have an alias, as per the original switch-case.

File: packages/composites/crypto-volatility-index/src/config/index.ts [hunk 3]
```diff
@@ -1,2 +1,13 @@
+import { Requester } from '@chainlink/ea-bootstrap'
+import { Config } from '@chainlink/types'
+
 export const DEFAULT_NETWORK = 'ETHEREUM'
 export const NAME = 'CRYPTO_VOLATILITY_INDEX'
+export const DEFAULT_ENDPOINT = 'volatility'
```
Comment (boxhock) [inline packages/composites/crypto-volatility-index/src/config/index.ts:6]: I think this endpoint name is misleading - suggest naming it "index"

**Review by boxhock — CHANGES_REQUESTED** [review]

File: packages/composites/bitcoin-json-rpc/src/config/index.ts (hunk 1 above)
Comment (bogos) [reply packages/composites/bitcoin-json-rpc/src/config/index.ts:4]: `getblockchaininfo` is the method that is executed. 'difficulty' is the default resulth path, but if this is changed, other information can be obtained. It seemed better to me to call the endpoint as the name of the method to execute. Do you think it's OK?

File: packages/composites/crypto-volatility-index/src/config/index.ts (hunk 3 above)
Comment (bogos) [reply packages/composites/crypto-volatility-index/src/config/index.ts:6]: we have 2 files in endpoint folder. The `index.ts` that exports all the methods defined in another file where we have the logic of the endpoints. So how about I call it `indexVolatility` ? to keep that file structure of `index.ts` to export everything.

Comment (boxhock) [reply packages/composites/crypto-volatility-index/src/config/index.ts:6]: Oooh, I see. Let's call it `volatilityIndex` then

**Review by boxhock — APPROVED** [review]

**Review by austinborn — APPROVED** [review]
